### PR TITLE
Redo slides for ERB

### PR DESCRIPTION
**Pull request by lxxxvi**
Here's a proposal for replacing the current slides/material for ERB in Lesson 16.

I'm looking forward to hearing your feedback.

### CODE REVIEW

File: slides/erb/index.html [hunk 1]
```diff
@@ -0,0 +1,348 @@
...
+            <pre><code class="hljs ruby">require "erb"
+template = File.read("./chores.html")
+ERB.new(template).result(binding)</code></pre>
+          </section>
+
+          <section>
+            <p>HTML Output</p>
+            <pre><code class="hljs html">&lt;h1&gt;Hi Bill 👋&lt;/h1&gt;
+
+&lt;p&gt;Don't forget to do following chores today:&lt;/p&gt;
```
Comment (nerdinand) [inline slides/erb/index.html:301]:
```suggestion
&lt;p&gt;Don't forget to do the following chores today:&lt;/p&gt;
```

not that you also have to fix the screenshot ;)

File: slides/erb/index.html [hunk 2]
```diff
@@ -0,0 +1,348 @@
...
+          <section>
+            <img class="no-border" src="images/chores.html.png" style="max-width: 50%;" />
+          </section>
+        </section>
+
+        <!-- Links to the relevant chapters in Ruby For Beginners -->
+        <section>
+          <h2>Additional Resources</h2>
+          <p>
+            <small><a href="https://http://webapps-for-beginners.rubymonstas.org/erb.html">Webapps For Beginners: Embedded Ruby</a></small>
```
Comment (nerdinand) [inline slides/erb/index.html:320]:
```suggestion
            <small><a href="http://webapps-for-beginners.rubymonstas.org/erb.html">Webapps For Beginners: Embedded Ruby</a></small>
```

File: slides/erb/index.html [hunk 3]
```diff
@@ -0,0 +1,348 @@
...
+          </section>
+
+
+          <section>
+            <p>Let's create an HTML template</p>
+
+            <p class="text-xl"><code>File ./chores.html</code></p>
+            <pre><code class="hljs html">&lt;h1&gt;Hi <%= username %> 👋&lt;/h1&gt;
+
+&lt;p&gt;Don't forget to do following chores today:&lt;/p&gt;
```
Comment (nerdinand) [inline slides/erb/index.html:265]: dito

File: slides/erb/index.html [hunk 4]
```diff
@@ -0,0 +1,348 @@
...
+            <p>You can write your presenter notes here</p>
+            <p>You start the presenter mode by pressing <kbd>S</kbd>.</p>
+            <p>By pressing <kbd>O</kbd> you'll see the slides overview.</p>
+          </aside>
+        </section>
+
+        <section>
+          <h2>Templating</h2>
+          <p>
+            We want to use Templating when we create output that contains <b>dynamic</b> content surrounded by <b>static</b> content.
```
Comment (nerdinand) [inline slides/erb/index.html:52]: But why do we want to use that? ;P Also, the word "template" may be unclear to some. Maybe an analogy with a word template for example would help?

File: slides/erb/index.html [hunk 5]
```diff
@@ -0,0 +1,348 @@
...
+        <section>
+          <h2>Interpolation?</h2>
+          <p>Why don't we use String interpolation?</p>
+          <pre><code class="hljs ruby">name = "Bill"
+"Hi #{name} 👋"         # => Hi Bill 👋</code></pre>
+
+          <p>💪</p>
+
+          <p class="fragment fade-in">
+            ✋ String interpolation gets messy if we have more complex outputs.
```
Comment (nerdinand) [inline slides/erb/index.html:75]: Maybe make an example for a messy string interpolation? E.g. a loop?

**Review by nerdinand — COMMENTED** [review]
Great stuff! I think this is a big leap forward in making this topic understandable. Great examples as usual! ^^

🚀

File: slides/erb/index.html (hunk 4 above)
Comment (lxxxvi) [reply slides/erb/index.html]: Yes, you're right! I have added another slide and restructured that area a bit.

File: slides/erb/index.html (hunk 5 above)
Comment (lxxxvi) [reply slides/erb/index.html:102]: Good idea! I have added the string interpolation implementation of the example to compare it.

**Comment by nerdinand** [comment]
Love it! Ship it! 
❤️ 🚢

**Review by nerdinand — APPROVED** [review]

**Pull request merged**